pages/task/schema/index.js

```diff
@@ -17,10 +19,14 @@ module.exports = (task) => {
     };
   };
 
+  const key = task.data.model;
+
   const options = task.nextSteps.map(nextStep => {
+    const option = content[key].fields.options.find(opt => opt.value === nextStep.id);
     return {
-      value: nextStep.id,
-      label: nextStep.id,
+      value: option.value,
+      label: option.label,
+      hint: option.hint,
```

Comment:
If we are using content like this, then we are doing something wrong. Radio groups and checkbox groups should lookup option labels and hints the same way as inputs lookup labels.